File: VB/10_Conversion/SetGlobalCustomFonts.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation

Namespace SetGlobalCustomFonts
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Set global custom fonts 
			Presentation.SetCustomFontsDirctory("..\..\..\..\..\..\Data\fonts")

			'Create a PPT document
			Dim ppt As New Presentation()

			'Load PPT file 
			ppt.LoadFromFile("..\..\..\..\..\..\Data\toPDF.pptx")

			'Save the PPT to PDF file format
			Dim result As String = "output.pdf"
			ppt.SaveToFile(result, FileFormat.PDF)

			Process.Start(result)
		End Sub
	End Class
End Namespace

File: CS/10_Conversion/SetGlobalCustomFonts.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;

namespace SetGlobalCustomFonts
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Set global custom fonts 
            Presentation.SetCustomFontsDirctory(@"..\..\..\..\..\..\Data\fonts");

            //Create a PPT document
            Presentation ppt = new Presentation();

            //Load PPT file 
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\toPDF.pptx");

            //Save the PPT to PDF file format
            String result = "output.pdf";
            ppt.SaveToFile(result, FileFormat.PDF);

            System.Diagnostics.Process.Start(result);
        }
    }
}